Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
And then what?  Silently disabling this?  We can't do that.  Log a warning?  I don't think that is the right behavior.